### PR TITLE
Fixes issue #32 Cannot read property 'currencyPair' of undefined

### PR DESCRIPTION
**Pull request by kevflynn**
This adds a check to ensure the ticker exists in the markets object. That being said - is there a reason we're not getting an updated list of markets/market tickets? Seems to me we should base this off of a dynamic list, not a static one that comes with the repo.

### CODE REVIEW

**Comment by dutu** [comment]
Until recently Poloniex did not include the `id` in the `returnTicker`, hence it was needed to work on a static list (it's still not documented).
Since it is now included the list of markets can be retrieved on the fly

File: lib/poloniex.js [hunk 1]
```diff
@@ -513,201 +513,219 @@ class Poloniex extends EventEmitter {
 
   // WebSocket API
   openWebSocket(options) {
-    this.wsVersion = options && options.version === 2 && 2 || 1;
-    switch(this.wsVersion) {
-      case 1: {
-        if (this.ws) {
-          this.ws.close();
+    
+    this.returnTicker().then(currencies => {
```
Comment (dutu) [inline lib/poloniex.js]: in case `returnTicker` returns an error, how should it behave?

Comment (kevflynn) [reply lib/poloniex.js]: good call. let me look into it

**Comment by kevflynn** [comment]
Sorr

**Pull request closed without merging**